docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -669,11 +672,11 @@ quarkus.http.auth.permission.login.paths=/google,/github
 quarkus.http.auth.permission.login.policy=authenticated
 ----
 
-If the endpoint is running on `http://localhost:8080`, you can also provide UI options for users to log in to either `http://localhost:8080/google` or `http://localhost:8080/github`, without having to add specific`/google` or `/github` JAX-RS resource paths.
+If the endpoint is running on `http://localhost:8080`, you can also provide UI options for users to log in to either `http://localhost:8080/google` or `http://localhost:8080/github`, without having to add specific `/google` or `/github` JAX-RS resource paths.
 Tenant identifiers are also recorded in the session cookie names after the authentication is completed.
 Therefore, authenticated users can access the secured application area without requiring either the `google` or `github` path values to be included in the secured URL.
 
-Default resolution can also work for Bearer token authentication but it might be less practical in this case because a tenant identifier will always need to be set as the last path segment value.
+Default resolution can also work for Bearer token authentication. Still, it might be less practical in this case because a tenant identifier must always be set as the last path segment value.
```

Comment:
```suggestion
Default resolution can also work for Bearer token authentication. Still, it might be less practical because a tenant identifier must always be set as the last path segment value.
```

Reply:
Updated. Thanks.